static/versionDetails.json

```diff
@@ -341,7 +341,7 @@
     },
     {
       "flag": "selinux",
-      "description": "This option sets the desired state of selinux, choices are enforcing, permissive, disabled.",
+      "description": "Sets the desired state of SELinux to either enforcing or permissive.",
```

Comment:
I'm not sure if this is the best description. We'll still set SELinux to disabled if it's currently permissive, yeah? (and of course you put the desired state as 'disabled')

Reply:
If you choose "disabled" we set it to permissive and configure it to be disabled on the next reboot. So I guess it is supported to disable it even though it's not immediately effective.